powershell-adapter/psDscAdapter/powershell.resource.ps1

```diff
@@ -3,13 +3,14 @@
 [CmdletBinding()]
 param(
     [Parameter(Mandatory = $true, Position = 0, HelpMessage = 'Operation to perform. Choose from List, Get, Set, Test, Export, Validate.')]
-    [ValidateSet('List', 'Get', 'Set', 'Test', 'Export', 'Validate', 'ClearCache')]
+    [ValidateSet('List', 'Get', 'Set', 'Test', 'Export', 'Validate', 'Schema', 'ClearCache')]
     [string]$Operation,
     [Parameter(Mandatory = $false, Position = 1, ValueFromPipeline = $true, HelpMessage = 'Configuration or resource input in JSON format.')]
     [string]$jsonInput = '@{}'
 )
 
-function Write-DscTrace {
```

Comment:
Because Pester requires the opening brace on the same line (as it's a parameter to a PowerShell function), so convention has been to keep the opening brace on the same line so it's consistent with Pester tests

Reply:
Should be resolved with the formatting setting in `settings.json`.